nixos/modules/security/pam.nix

```diff
@@ -18,6 +104,28 @@ let
         description = lib.mdDoc "Name of the PAM service.";
       };
 
+      rules = mkOption {
+        # This option is experimental and subject to breaking changes without notice.
+        visible = false;
+
+        description = lib.mdDoc ''
+          PAM rules for this service.
+
+          ::: {.warning}
+          This option and its suboptions are experimental and subject to breaking changes without notice.
```

Comment:
What is the purpose if the options are not visible? The descriptions are not rendered in the manual. My thought was to make the warning prominent in the source code, since that's the only place these options can be seen.

Reply:
Ah I see. I guess it's fine then as is.